### ISSUE TITLE
GlassCast returns null for file field

### ISSUE THREAD

**Issue by trnktms** [mikeedwards83/Glass.Mapper#184]
Hi,

the problem if I use the GlassCast method that it returns null for the file field if the file item is not exists in the current language only in Preview Mode and Normal Mode. In Page Editor Mode it works fine.

Cast method call:
![image](https://cloud.githubusercontent.com/assets/15426455/10843939/5cff1b30-7efc-11e5-851a-fbbca0330ef9.png)

Generated type:
![image](https://cloud.githubusercontent.com/assets/15426455/10843988/b43d3f26-7efc-11e5-9851-47b8d3deaf8a.png)

Thank you and regards,
Tamás

**Comment by mikeedwards83** [comment]
This might help:

http://www.glass.lu/Mapper/Sc/Documentation/VersionCountDisabler

**Comment by trnktms** [comment]
Thanks for the quick response. Yes it works, but I think we need to use this only for the 1 file field, not to the whole item.
![image](https://cloud.githubusercontent.com/assets/15426455/10845278/cc41a7f6-7f06-11e5-995a-3a16e9947438.png)
This means that I disabled for the whole item casting. I think it's general how sitecore image and file types should work.

**Comment by mikeedwards83** [comment]
Have you marked the field as shared in Sitecore?

**Comment by trnktms** [comment]
The field shouldn't be shared in the template, because the value of the field can be different per languages. I just would like to avoid to create media items in all langauges.

**Comment by trnktms** [comment]
Hi,

I checked the code and I found this:
https://github.com/mikeedwards83/Glass.Mapper/blob/ba281c65db5438ab746a0b96e6f0b35d908ed8b6/Source/Glass.Mapper.Sc/Utilities.cs - public static bool DoVersionCheck(Config config)
-> we can't force it to show in preview/normal mode because of "if (config != null && config.ForceItemInPageEditor && GlassHtml.IsInEditingMode)" GlassHtml.IsInEditingMode only true in Page Editor.

![image](https://cloud.githubusercontent.com/assets/15426455/13011256/38a634a6-d1a6-11e5-8164-52b49e29da6f.png)


I don't know that is it a bug or designed like this.

Thank you for your reponse!
